charts/kyverno/README.md

```diff
@@ -48,6 +48,8 @@ $ helm install kyverno --namespace kyverno kyverno ./charts/kyverno
 
 The command deploys Kyverno on the Kubernetes cluster with default configuration. The [installation](https://kyverno.io/docs/installation/) guide lists the parameters that can be configured during installation.
 
+Kyverno ClusterRoles/ClusterRoleBindings name must have suffix `:webhook`. Example, `*:webhook`.
```

Comment:
"Kyverno ClusterRole/ClusterRoleBinding names must have the suffix `:webhook`. Ex., `*:webhook`.

Reply:
Suggest clarifying that this is only required for the ClusterRole/ClusterRoleBinding that manages the webhook configurations. Other clusterroles names are configurable.